qutip/orbital.py

```diff
@@ -1,7 +1,7 @@
 __all__ = ['orbital']
 
 import numpy as np
-from scipy.special import factorial
+from scipy.special import factorial, sph_harm
```

Comment:
`factorial` is no longer used.
```suggestion
from scipy.special import sph_harm
```